packages/flutter/test/material/floating_action_button_theme_test.dart

```diff
@@ -191,7 +216,8 @@ void main() {
     expect(_getIconSize(tester).height, iconSize);
   });
 
-  testWidgets('FloatingActionButton.extended uses custom properties when specified in the theme', (WidgetTester tester) async {
+  testWidgets('Material3: FloatingActionButton.extended uses custom properties when specified in the theme', (WidgetTester tester) async {
+    const ColorScheme colorScheme = ColorScheme.light();
```

Comment:
Seems we can remove this variable or use it in line 231 to fix the Linux analyze test